packages/css/src/button/button.scss

```diff
@@ -27,16 +27,7 @@
   }
 }
 
-// Specificity hack to be able to style secondary button focus state
-// This isn't currently possible with the Utrecht component
-// We should really pull this component in our own repo...
-.amsterdam-button--secondary.amsterdam-button--secondary {
-  &:focus {
-    color: var(--amsterdam-button-secondary-focus-color);
-  }
-}
-
-.amsterdam-button--secondary:focus:not(:disabled, [aria-disabled="true"]) {
+.amsterdam-button--secondary:focus:not(:hover, :disabled, [aria-disabled="true"]) {
```

Comment:
I think we can get rid of the `:disabled` selector. I'm actually not quite sure what the `[aria-disabled="true"]` selector does here, or how `aria-disabled` should be styled. 

We do need the `:not(:hover)` selector here. This rule sets a 2px box shadow, and line 26 sets a 3px box shadow. If we don't explicitly set a 2px box shadow here, the default one will be removed because of styling in the Utrecht button